packages/slate/src/create-editor.ts

```diff
@@ -135,6 +135,15 @@ export const createEditor = (): Editor => {
       }
     },
 
+    getFragment: () => {
```

Comment:
Second new piece of code allowing just the fragment to be changed, if you don't need to mess with the dom copying stuff. (This is actually the only piece I need for my specific use-case, but it seemed more generally useful to also allow pluggability of the entire dataTransfer.)